.golangci.yaml

```diff
@@ -119,23 +119,23 @@ linters:
     - gocyclo
     - gofmt
     - gofumpt
-    - golint
+    # - golint
     - gomnd
     - gomodguard
     - goprintffuncname
     - gosec
     - gosimple
     - govet
     - grouper
-    - ifshort
+    # - ifshort
     - importas
     - ineffassign
     - interfacebloat
     - lll
     - logrlint
     - maintidx
     - makezero
-    - maligned
+    # - maligned
```

Comment:
This is replaced by govet's fieldalignment and govet is already enabled.

Reply:
fieldalignment is not enabled by default with [govet](https://golangci-lint.run/usage/linters/#govet), there are a few linter errors popping up once enabled though. So not sure where maligned failed.